test/SlowTests/Issues/RavenDB_18163.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Queries.Timings;
+using Raven.Server;
+using Raven.Server.Documents;
+using Raven.Server.Documents.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Raven.Tests.Core.Utils.Entities;
+using Raven.Server.Config;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_18163 : RavenTestBase
+    {
+        public RavenDB_18163(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CreatingARevisionManuallyAfterEnablingRevisionsForAnyCollection()
```

Comment:
Need another test to check if `ForceRevisionCreation` is working when used in the same session.
```
using (var session = store.OpenSession())
{
    var company = new Company { Name = "HR" };
    session.Store(company);
    session.Advanced.Revisions.ForceRevisionCreationFor<Company>(company);
    companyId = company.Id;
    session.SaveChanges();
}

```

Reply:
Done

Reply:
https://github.com/ravendb/ravendb/blob/1af03d80c59f9c872a4c16d6ce053004c9127dca/test/SlowTests/Issues/RavenDB_18163.cs#L82